src/lib/scene-internal.ts

```diff
@@ -646,6 +657,90 @@ export class SceneInternal implements Renderer {
     this.queueDraw();
   }
 
+  /**
+   * Initialize the scale and offset of the Scene if possible. If the canvas has
+   * zero width or height, then the scale and offset will not be initialized.
+   */
+  private initView() {
+    if (this.isViewInitialized) {
+      return;
+    }
+
+    const bounds = this.canvas.getBoundingClientRect();
+    const width = bounds.right - bounds.left;
+    const height = bounds.bottom - bounds.top;
```

Comment:
`DOMRect` has `width` and `height` props, shouldn't need to recompute here. 

```suggestion
    const {width, height} = this.canvas.getBoundingClientRect();
```